openmdao/core/group.py

```diff
@@ -53,6 +53,10 @@ class Group(System):
         Dictionary of input_name: (output_name, src_indices) connections.
     _group_inputs : dict
         Mapping of promoted names to certain metadata (src_indices, units).
+    _static_group_inputs : dict
+        Group inputs added outside of setup/configure.
```

Comment:
Where else can we add inputs? Does this just mean after prob.setup?

Reply:
Static means everything outside of system setup and configure.  If things are added in setup/configure they're considered 'dynamic' and they get blown away if problem setup is called again.  Static stuff is never removed.

Reply:
this would be a nice comment to put in the code somewhere

Reply:
updated comment